frontend/src/components/DepositWithdrawView.tsx

```diff
@@ -31,6 +31,8 @@ export const DepositWithdrawView: React.FC<DepositWithdrawViewProps> = ({ tokenB
       } else {
         await withdrawRawAmount(vault.id, MOCK_ERC_20_ADDRESS, address, amount, signer)
       }
+      window.location.reload()
```

Comment:
Instead of reloading the page, would it make more sense to reload only the balances, i.e. calling `getTokenBalances` to refresh the props?

Also, we should have a better system for timing the refresh, because it should happen after the transaction has been completed

Reply:
yeah thats a needed optimization :)